### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-04-28-dashboard.md [hunk 1]
```diff
@@ -12,5 +12,6 @@ authors:
...
+- **Sankey Chart**. Our new **Sankey diagram** feature, which can you find under the **Categorical** panel type, is built using `fromstate` and `tostate` fields in your query. The Sankey diagram helps you understand the flow of the log events within a distributed system. The width of an arrow or stripe in the Sankey diagram shows the proportion of a quantity.
 - **Missile Map**. Our new **Missile Map** feature, which you can find under the **Map** panel type, provides visibility into the geographic origins of threats and their target points. By hovering over each line, you can find the threat name, latitude/longitude details of threat origin, and latitude/longitude details of threat target point.
+- **Box Plot Charts**. Our new **Box Plot Charts** feature, which you can find under the **Time Series** and **Categorical** panel type graphically depicts groups of data using quartiles, and this is built by including `_min`, `_pct_25`, `_pct_50`, `_pct_75`, and `_max` in your query.
```
Comment (JainM6) [inline blog-service/2023-04-28-dashboard.md:17]:
```suggestion
- **Box Plot Charts**. Our new **Box Plot Charts** feature, which you can find under the **Time Series** and **Categorical** panel type, graphically depicts groups of data using quartiles and is built by including `_min`, `_pct_25`, `_pct_50`, `_pct_75`, and `_max` in your query. 
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Minor formatting changes.

File: blog-service/2023-04-28-dashboard.md [hunk 2]
```diff
@@ -12,5 +12,6 @@ authors:
 
 We're excited to announce the release of several new features in the **Dashboards (New)**:
 
-- **Sankey Chart**. Our new **Sankey diagram** feature, which can you find under the **Categorical** panel type, is built using two fields in your query, such as `fromstate` and `tostate`. The Sankey diagram helps you understand the flow of the log events within a distributed system. The width of an arrow or stripe in the Sankey diagram shows the proportion of a quantity.
+- **Sankey Chart**. Our new **Sankey diagram** feature, which can you find under the **Categorical** panel type, is built using `fromstate` and `tostate` fields in your query. The Sankey diagram helps you understand the flow of the log events within a distributed system. The width of an arrow or stripe in the Sankey diagram shows the proportion of a quantity.
```
Comment (JainM6) [inline blog-service/2023-04-28-dashboard.md:15]:
```suggestion
- **Sankey Chart**. Our new **Sankey diagram** feature, which you can find under the **Categorical** panel type, is built using the `fromstate` and `tostate` fields in your query. The Sankey diagram helps you understand the flow of the log events within a distributed system. The width of an arrow or stripe in the Sankey diagram shows the proportion of a quantity.
```

Comment (JainM6) [reply blog-service/2023-04-28-dashboard.md:15]: It wasn't resolved.

**Review by JainM6 — DISMISSED** [review]
Changes were not implemented.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**